app.py

```
@@ -27,24 +28,36 @@ def resource_path(relative_path):
+# Configure logging
+logging.basicConfig(
+    level=logging.DEBUG,
+    format='%(asctime)s - %(levelname)s - %(message)s',
+    handlers=[
+        logging.FileHandler('rsteye.log'),
+        logging.StreamHandler()
+    ]
+)
+
 class RstEyeApp:
     def __init__(self, image_path, interval=POPUP_INTERVAL * 60, fullscreen=False):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)